pgxn/neon/walproposer.h

```diff
@@ -707,12 +707,11 @@ typedef struct WalProposerConfig
 	char	   *neon_timeline;
 
 	/*
-	 * Comma-separated list of safekeepers, in the following format:
-	 * host1:port1,host2:port2,host3:port3
+	 * Comma-separated list of safekeeper connection strings
```

Comment:
Maybe it's worth explaining (quickly) why comma-separated list of connection strings make sense here also?